### ISSUE TITLE
[JS] Selected state not reflected for comboboxes in WebChat

### ISSUE THREAD

**Issue by paulcam206** [microsoft/AdaptiveCards#4027]
Per email thread:

> Combo box for state selection does not announce “Selected” state. If screen reader user selects “Washington”, it only announces “Washington” instead of “Washington, selected.”
